app/webrtc/server/methods/webrtcCreateSession.js

```diff
@@ -0,0 +1,43 @@
+import { Meteor } from 'meteor/meteor';
+import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
+
+import { Messages, Users } from '../../../models/server';
+import { callbacks } from '../../../callbacks/server';
+import { canSendMessage } from '../../../authorization/server';
+import { SystemLogger } from '../../../logger/server';
+
+Meteor.methods({
```

Comment:
IMO we should avoid using meteor methods and build these type of functionality using endpoints

Reply:
In noticed we already have an endpoint defined [here](https://github.com/RocketChat/Rocket.Chat/blob/2fa2ba0fdd6cd29fed29c1b1118ce89e54d5856e/app/livechat/server/api/v1/videoCall.js#L10) for starting a call so IMO this is a good time to modify that endpoint as well to support webrtc calls and use the same on the UI

Reply:
@djcruz93 Any updates on this part?